src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -308,11 +308,14 @@ class DocAnnotator extends Annotator {
             thread = new DocDrawingThread(threadParams);
         } else if (type === TYPES.point) {
             thread = new DocPointThread(threadParams);
-        } else {
```

Comment:
Maybe instead of an error (or getting rid of that error altogether) we can do `this.emit('annotationerror', __('annotations_create_error'));` and return, which will trigger a notification in Preview informing the user that their annotation failed to create?